gax/test/unit/streaming.ts

```diff
@@ -294,6 +299,72 @@ describe('streaming', () => {
       done();
     });
   });
+  it('cancels in the middle when new retries are enabled', done => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    function schedulePush(s: any, c: number) {
```

Comment:
There's no stream type?

Reply:
good question - this test was copied, comments and all, from another one. Instead of "any", I have modified to do some icky typecasting from PassThrough to unknown to CancellableStream. Not the best, but not the worst. It's at least a little more specific.